erpnext/support/doctype/issue/issue.js

```diff
@@ -1,13 +1,53 @@
 frappe.ui.form.on("Issue", {
 	onload: function(frm) {
 		frm.email_field = "raised_by";
+
 		if (frm.doc.service_level_agreement) {
-			set_time_to_resolve_and_response(frm);
+			frappe.call({
```

Comment:
why 2 separate calls? Get all related information in one call

Reply:
Reverted to one call to get data